### PR TITLE
Remove Unused Components And Helpers

### PR DESCRIPTION
**Pull request by Ap0c**
# Why are you doing this?

With the final contributions page migrated over to Circles in #621, we can delete some stuff! This PR removes some unused components and helpers. Currently the base branch for this is the aforementioned Circles PR. I'll change it to master once that goes out (or we could merge this into that).

cc @JustinPinner 

# Changes

- Deleted the `Bundle` and `ContribAmounts` components.
- Deleted some helper types and functions that are no longer in use.

### CODE REVIEW

File: assets/helpers/contributions.js [hunk 1]
```diff
@@ -526,14 +440,10 @@ export {
   circlesParse,
   billingPeriodFromContrib,
   errorMessage,
-  getContribKey,
```
Comment (svillafe) [inline assets/helpers/contributions.js]: I thought this list would become much shorter, are we using all the remaining functions somewhere ins the codebase?

Comment (Ap0c) [reply assets/helpers/contributions.js]: Yes, combination of the new contributions component and the checkout pages.

Comment (svillafe) [reply assets/helpers/contributions.js]: Ok, If we are looking into rebuilding the checkout page, we should probably need to check whether is possible to simplify the number of functions here. For example, I thought that `circlesParse` would become `parse` now that everything is circles.

Comment (Ap0c) [reply assets/helpers/contributions.js]: Yup, that's something that will be possible if we do a rebuild of the checkout pages.

**Pull request merged**